spec/features/user_past_order_view_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+describe 'user past order views', type: :feature do
+  let(:user) { Fabricate(:user) }
+
+  before(:each) do
+    ApplicationController.any_instance.stubs(:current_user).returns(user)
+    visit user_path(user)
+    page.click_link "Past Orders"
+  end
+
+  it 'shows past orders' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.